### PR TITLE
Document change for Observability 1.5

### PR DESCRIPTION
**Pull request by baoy1**
# Description
Documentation update for Observability 1.5 which includes:

- Update of support matrix
- Add document for PowerMax, metrics, installer, etc
- Release note update

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/583|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. No edits are required.

**Review by P-Cao — DISMISSED** [review]

**Review by taohe1012 — APPROVED** [review]

**Comment by baoy1** [comment]
Hi all, just now I updated `CSM Modules Support Matrix for Dell CSI Drivers ` to support CSM 2.6 and Obs 1.5. Please take a look and re-approve if no questions. Thanks!
image

**Review by P-Cao — DISMISSED** [review]

**Comment by baoy1** [comment]
Hi all, just now I fixed dead link "CSI PowerMax driver with Proxy in StandAlone mode." in observability doc. Please take a look and re-approve if no questions. Thanks!

**Review by P-Cao — APPROVED** [review]

**Review by taohe1012 — DISMISSED** [review]

**Review by forrestxia — DISMISSED** [review]

**Review by taohe1012 — DISMISSED** [review]

**Review by taohe1012 — APPROVED** [review]

**Review by forrestxia — APPROVED** [review]

File: content/docs/observability/deployment/offline.md [hunk 1]
```diff
@@ -164,13 +166,39 @@ To perform an offline installation of a Helm chart, the following steps should b
     [user@anothersystem /home/user/offline-karavi-observability-bundle/helm]# kubectl get secret karavi-authorization-config proxy-server-root-certificate proxy-authz-tokens -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | sed 's/name: karavi-authorization-config/name: isilon-karavi-authorization-config/' | sed 's/name: proxy-server-root-certificate/name: isilon-proxy-server-root-certificate/' | sed 's/name: proxy-authz-tokens/name: isilon-proxy-authz-tokens/' | kubectl create -f -
     ``` 
 
+    CSI Driver for PowerMax:
```
Comment (P-Cao) [inline content/docs/observability/deployment/offline.md:169]: Does the offline installation need the same modification about copying configmap & secret?

Comment (baoy1) [reply content/docs/observability/deployment/offline.md:169]: We cannot predict the customized name so I think no need to update offline installation, but that is better to add some comment in the script to explain offline script excepts default names.

Comment (baoy1) [reply content/docs/observability/deployment/offline.md:169]: I will update csm-docs to add the comment to online installer, and for offline we can update the command.

Comment (baoy1) [reply content/docs/observability/deployment/offline.md]: Updated. Please have a check.

**Review by forrestxia — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by shanmydell — APPROVED** [review]

**Review by P-Cao — APPROVED** [review]

**Pull request merged**